src/deviceAPI/process/stream/image/createFromFile.js

```diff
@@ -0,0 +1,36 @@
+const { createDataUriFromFile } = require('../../../../utils/imageUtils')
+const serializeAxiosResponse = require('../../../../utils/serializeAxiosResponse')
+const { DEFAULT_REQ_OPTS } = require('../../../../utils/defaultReqOpts')
+const AuthorizedDeviceAPI = require('../../../AuthorizedDeviceAPI')
+
+/**
+ * Uploads a process image based given filen path
+ * for information on the necessary
+ * parameters check the api documentation in the readme *
+ * @async
+ * @param processId the process Id
+ * @param streamId the stream Id
+ * @param body Object containing the image and metadata
+ * @param body.pathToImage the absolute path to the image
+ * @param body.imageCreatedAt the unix timestamp of the image creation
```

Comment:
maybe provide a unit for the timestamp here (we always use milliseconds) because it's no totally clear from the example on line 19